packages/react-core/src/Authenticator/hooks/useAuthenticatorRoute/__tests__/utils.spec.ts

```diff
@@ -71,17 +72,20 @@ describe('getRouteMachineSelector', () => {
   it.each([
     [
       'confirmResetPassword',
-      [...commonSelectorProps, resendCode, validationErrors],
+      [...commonSelectorProps, resendCode, validationErrors, route],
```

Comment:
I was thinking about adding `route` to `commonSelectorProps`, but that started making an assumption on order of dependencies, so I just left route at the end of the array to mimic the implementation below.

Reply:
Thanks for adding a test!!